chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -125,7 +125,8 @@ const setPreChatAndInitiateChat = async (facadeChatSDK: FacadeChatSDK, dispatch:
             eventName: BroadcastEvent.MaximizeChat,
             payload: {
                 height: state?.domainStates?.widgetSize?.height,
-                width: state?.domainStates?.widgetSize?.width
+                width: state?.domainStates?.widgetSize?.width,
+                runtimeId : TelemetryManager.InternalTelemetryData.lcwRuntimeId
```

Comment:
null check with optional chaining is used in the previous location, is it also needed here?

<img width="873" height="748" alt="image" src="https://github.com/user-attachments/assets/be4220a8-4659-457e-88b8-e87cbda2fd10" />